db/migrate/20160426145517_move_host_description_to_host_comment.rb

```diff
@@ -0,0 +1,29 @@
+class MoveHostDescriptionToHostComment < ActiveRecord::Migration
+  class System < ActiveRecord::Base
+    self.table_name = "katello_systems"
+  end
+
+  def up
+    System.find_each do |system|
```

Comment:
The reason I thought to do it this way was because this will only appear on hosts that were once systems but I can change it if you prefer Host.